bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -133,6 +133,31 @@ <h3>Tracking Protection</h3>
     <li>shavar.services.mozilla.com</li>
   </ul>
 
+  <h2 id="pocket"><b>Pocket</b></h2>
+
+  <p>Pocket pays out bounties at a reduced rate and only if the reported bug has Critical or High severity rating. The scope for Pocket in this program is limited to the below targets. Any API endpoint or path not explicitly mentioned is considered out of scope.</p>
```

Comment:
```suggestion
  <p>Pocket pays out bounties at a lower rate than Mozilla critical sites and only if the reported bug has a Maximum or High risk level. The scope for Pocket in this program is limited to the targets below. Any API endpoint or path not explicitly mentioned is considered out of scope.</p>
```

Reply:
If by "reduced rate" you mean a dollar amount lower than critical (or core, or other) sites, we should say that.

Reply:
I'll make that change to clarify "reduce rate", regarding the other changes, I prefer we use critical and high severity rating to match what we have in our guidelines, https://wiki.mozilla.org/Security_Severity_Ratings/Web, and the ratings we have on bugzilla, sec-critical, sec-high and so on

Reply:
Aha! I didn't know about that page, that makes sense. Sounds good.